src/main/kotlin/com/exactpro/th2/codec/AbstractCodecProcessor.kt

```diff
@@ -42,7 +43,9 @@ abstract class AbstractCodecProcessor(
     }
 
     protected fun String?.onErrorEvent(message: String, messagesIds: List<MessageID> = emptyList(), cause: Throwable? = null) {
-        logger.error(message, cause)
+        logger.error("$message. Message id's: ${messagesIds.joinToString(", ") { 
```

Comment:
Use a lambda to generate a message instead of a String to do nothing when the log level of the app is below then the record